src/admin/components/website/modalParts/WebsiteFeedbackEmailOverride.tsx

```diff
@@ -0,0 +1,139 @@
+//
+// LESERKRITIKK v2 (aka Reader Critics)
+// Copyright (C) 2017 DB Medialab/Aller Media AS, Oslo, Norway
+// https://github.com/dbmedialab/reader-critics/
+//
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the GNU General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option) any later
+// version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program. If not, see <http://www.gnu.org/licenses/>.
+//
+
+import * as React from 'react';
+import { InputError } from 'admin/components/form/InputError';
+import { connect } from 'react-redux';
+import Validator from 'admin/services/Validation';
+
+class WebsiteFeedbackEmailOverride extends React.Component <any, any> {
+	private readonly validator : Validator;
+
+	constructor (props) {
+		super(props);
+		this.state = {
+			value: '',
+			touched: false,
+		};
+
+		this.validator = new Validator();
+
+		this.onDelete = this.onDelete.bind(this);
+		this.onSubmit = this.onSubmit.bind(this);
+		this.onKeyPress = this.onKeyPress.bind(this);
+		this.onEdit = this.onEdit.bind(this);
+		this.checkValidation = this.checkValidation.bind(this);
+	}
+
+	checkValidation () {
+		const {value} = this.state;
+		const isEmail = this.validator.validate('feedbackEmailOverride', value, { required: true });
+
+		if (isEmail.isError) {
+			return isEmail.message;
+		}
+		else {
+			const isUnique = this.validator.validate('uniqueness',
+				this.props.feedbackEmailOverride.concat(value));
+
+			if (isUnique.isError) {
+				return isUnique.message;
+			}
+
+			return false;
+		}
+	}
+
+	onDelete (index) {
+		const {feedbackEmailOverride: override} = this.props;
+		if (index >= 0) {
+			const feedbackEmailOverride = override.asMutable();
+			feedbackEmailOverride.splice(index, 1);
```

Comment:
use pop(index) if you only delete 1 element by index

Reply:
pop remove only last element, not by index